src/App.tsx

```diff
@@ -1,61 +1,119 @@
 import './App.scss';
+import { useState } from 'react';
+import { TodoList } from './components/TodoList';
+import { User } from './types/User';
+import { Todo } from './types/Todo';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+function getUser(userId: number): User | null {
+  const findUser = usersFromServer.find(user => user.id === userId);
+
+  return findUser || null;
+}
+
+const todos = todosFromServer.map((todo) => {
+  return (
+    {
+      ...todo,
+      user: getUser(todo.userId),
+    }
+  );
+});
```

Comment:
you can simplify the code to:
```suggestion
const todos = todosFromServer.map((todo) => ({
  ...todo,
  user: getUser(todo.userId),
});
```